client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -107,7 +107,10 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
     mostRecentCredentials,
   } = useLogin(setCookie);
   const getUserPermissions = useGetUserPermissions();
-  const { refreshToken } = useRefreshToken();
+  const { refreshToken } = useRefreshToken(() => {
+    Cookies.remove('auth');
+    setError(AuthError.Timeout);
```

Comment:
I think you should add  `setCookie(undefined);` here too.

this logs the user out, but does not prompt  - I have to navigate to a different page before I'm told that I'm logged out.
two options:
- navigate to the login page, with a message displayed 'You've been logged out due to inactivity' or similar
- do something here to update the local state and force a refresh which would refresh `AuthenticationAlert` and prompt the user

I rather like option 1 even though we have a beautiful 'you've been logged out' alert already and redirecting to login is slightly unconventional.

for the second option, a dirty version would be to add `window.location.reload();` to the callback method here 😉

Reply:
> this logs the user out, but does not prompt - I have to navigate to a different page before I'm told that I'm logged out.

Not quite getting this one. I get the correct (same as current) dialog that pops up at logout time:
<img width="779" alt="Screenshot 2023-11-21 at 10 25 11 AM" src="https://github.com/msupply-foundation/open-msupply/assets/5456533/8813025e-0c41-49a4-98c5-3d647eb889d5">

I get this without having to navigate anywhere else, it just appears. And this works whether I have `setCookie(undefined)` or not -- are you seeing something different?

Reply:
yeah - I get the console log saying that things have been cleared, but nothing shows in the UI. I left it for a further 5 mins.. nothing. I navigated to another page and it popped up.

Reply:
Hmm, interesting, I'll check it out in different browsers and compare it to the current behaviour.

I thought `setError(AuthError.Timeout)` should cause this prompt. Does it work as expected for you in the current system?

Reply:
Yeah, I can't replicate the problem you're describing in either Firefox or Chrome. I'll wait and see what it looks like on yours tomorrow when we're both in the office. Damn, thought this was sorted :/

Reply:
I've tested again and it worked fine that time. let's just merge this and monitor.. what you've done makes sense and is much better than it is currently

Reply:
Good to hear! Will keep an eye on it.